pkg/storage/backend-storage/backend-storage.go

```diff
@@ -413,6 +417,30 @@ func (bs *BackendStorage) getStorageClass() (string, error) {
 	}
 }
 
+func (bs *BackendStorage) getBackendPVCSize(storageClassName string) (string, error) {
+	// Default size is 10Mi if the annotation is not set
+	size := DefaultPVCSize
+
+	obj, exists, err := bs.scStore.GetByKey(storageClassName)
+	if err != nil || !exists {
+		return size, err
+	}
+
+	sc := obj.(*storagev1.StorageClass)
+	if val, ok := sc.Annotations[MinimumSupportedPVCSize]; ok {
+		if parsedSize, parseErr := resource.ParseQuantity(val); parseErr == nil {
+			defaultSize, _ := resource.ParseQuantity(size)
```

Comment:
why not work with parsed values and return the parsed quantity instead of string? since you parse it again in the calling function.

Reply:
ack